sklearn/datasets/tests/test_openml.py

```diff
@@ -215,21 +217,21 @@ def test_fetch_openml_iris(monkeypatch):
 
     _monkey_patch_webbased_functions(monkeypatch, data_id, test_gzip)
     assert_warns_message(
-        UserWarning,
-        "Multiple active versions of the dataset matching the name"
-        " iris exist. Versions may be fundamentally different, "
-        "returning version 1.",
-        _fetch_dataset_from_openml,
-        **{'data_id': data_id, 'data_name': data_name,
-           'data_version': data_version,
-           'target_column': target_column,
-           'expected_observations': expected_observations,
-           'expected_features': expected_features,
-           'expected_missing': expected_missing,
-           'expect_sparse': False,
-           'expected_data_dtype': np.float64,
-           'expected_target_dtype': object,
-           'compare_default_target': True}
+         UserWarning,
+         "Multiple active versions of the dataset matching the name"
+         " iris exist. Versions may be fundamentally different, "
+         "returning version 1.",
+         _fetch_dataset_from_openml,
+         **{'data_id': data_id, 'data_name': data_name,
+            'data_version': data_version,
+            'target_column': target_column,
+            'expected_observations': expected_observations,
+            'expected_features': expected_features,
+            'expected_missing': expected_missing,
+            'expect_sparse': False,
+            'expected_data_dtype': np.float64,
+            'expected_target_dtype': object,
+            'compare_default_target': True}
```

Comment:
In all the above, changes are due to fixing the merge. We don't want to see any of these changes in this diff because it would complicate e.g. git blame. So the easiest to remove these changes would be,
 - either manually with `git diff master...HEAD` (or `HEAD...master` I never can remember) and editing the file until none of it shows.
 - or copying this file somewhere, reverting to the version on master `git checkout master -- <path_to_this_file>` than adding back your changes.
Don't worry about individual commits as in the end everything will be squashed into a single one, so only the total diff matter. Apart for this this looks good.